tests/Validation/ValidationValidatorTest.php

```diff
@@ -2585,6 +2585,7 @@ public function testValidateImplicitEachWithAsterisksConfirmed()
         $this->assertFalse($v->passes());
         $this->assertTrue($v->messages()->has('foo.0.password'));
         $this->assertTrue($v->messages()->has('foo.1.password'));
+        $this->assertTrue($v->messages()->has('foo.*.password'));
```

Comment:
Yes, it's related. The above test causes the `\ErrorException` without this PR. Did I miss a better place for it?